### PR TITLE
Fix npm run lint script

### PR DESCRIPTION
**Pull request by pdehaan**
r? @gregglind

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,13 +62,13 @@
   "scripts": {
     "build": "bash ./bin/xpi.sh",
     "eslint": "eslint addon test --ext jsm --ext js --ext json",
+    "firefox": "export XPI=dist/linked-addon.xpi && npm run build && node run-firefox.js",
+    "harness_test": "export XPI=dist/linked-addon.xpi && mocha test/functional_tests.js --retry 2 --reporter json",
     "lint": "npm-run-all lint:*",
     "lint:addons-linter": "# actually a post build test:  bin/addonLintTest ' + require('./package.json').name",
-    "lint:eslint": "npm eslint",
+    "lint:eslint": "npm run eslint",
```
Comment (pdehaan) [inline package.json]: This was the only changed line. I think the other 2 lines were just some auto A-z sorting by npm.

Comment (gregglind) [reply package.json]: Thanks!  Maybe look over the main eslintrc, and see if you think that's a good set of stuff, or if there is more to add?

Comment (pdehaan) [reply package.json]: Done. Beefier ESLint proposal submitted in #37 PR.

**Comment by motin** [comment]
Good catch. To avoid merge conflicts, let's approve #37 first, then #41 (which includes the change of this PR)

**Comment by motin** [comment]
Please close this PR, since #41 is already merged and included the change described in this PR (https://github.com/mozilla/shield-studies-addon-template/pull/41/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L67)

**Comment by biancadanforth** [comment]
Closing, since I merged #37 , per [this discussion](https://github.com/mozilla/shield-studies-addon-template/pull/49#issuecomment-365979438).

**Pull request closed without merging**